cpp/src/io/reader.cpp

```diff
@@ -195,9 +195,11 @@ std::vector<Node> Reader::GetNodesWithinBox(const Box &box, double min_resolutio
 
     auto header = GetLasHeader();
     auto copc_info = GetCopcHeader();
+    auto max_depth = GetDepthAtResolution(min_resolution);
```

Comment:
what did we say on this? I didn't think i agreed to min_resolution 😆

Reply:
I thought we did agree on min hahaha

Because it'll be at least this much resolution, so it's the minimum resolution or better.

Reply:
yep